### PR TITLE
Add sample usage for light SKU

### PR DESCRIPTION
**Pull request by jpiyali**
Adding sample page for usage of Light SKU

### CODE REVIEW

File: testlightsku.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        </head>
+    <br/>
+    <script type="text/javascript">
+        var appInsights = new Microsoft.ApplicationInsights({instrumentationKey: "INSTRUMENTATION_KEY"});
+    </script>
+<script>
+    function onPageViewClick() {
+
+        var pageView = {
+            name: "Microsoft.ApplicationInsights.INSTRUMENTATION_KEY.Pageview",
```
Comment (markwolff) [inline testlightsku.html:19]: We remove the dashes from this ikey. Not sure if it will actually impact telemetry though

File: testlightsku.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            tags: [ { "device.id": "AABA40BC-EB0D-44A7-96F5-ED2103E47AE9"} ],
+            data: {
+                "custom data": {
+                    "data1": "value1"
+                }
+            },
+            baseType: "PageviewData",
+            baseData: { 
+                name: "Test Page",
+                url: "http://localhost:8080/testlightsku.html"
```
Comment (markwolff) [inline testlightsku.html:32]: Should this be `uri`?

https://github.com/Microsoft/ApplicationInsights-JS/blob/master/AppInsightsCommon/src/Interfaces/IPageViewTelemetry.ts#L16

Comment (jpiyali) [reply testlightsku.html:32]: yeah, this should be uri.  Fxing

File: testlightsku.html (hunk 1 above)
Comment (jpiyali) [reply testlightsku.html:19]: Will add text

**Pull request merged**